sentences/nl/light_HassLightSet.yaml

```diff
@@ -3,19 +3,22 @@ intents:
   HassLightSet:
     data:
       - sentences:
-          - "[<doe>|<zou>] <name>[ ][<helderheid>] [<naar>] <brightness> [willen|kunnen] [<doe>]"
-          - "[<doe>|<zou>] <helderheid> [van] <name> [<naar>] <brightness> [willen|kunnen] [<doe>]"
+          - "[<doe>|<zou>] <name>[ ][<lamp>][ ][<helderheid>] [<naar>] <brightness> [willen|kunnen] [<doe>]"
+          - "[<doe>|<zou>] <helderheid> [van] <name>[ ][<lamp>] [<naar>] <brightness> [willen|kunnen] [<doe>]"
         response: "brightness"
+
```

Comment:
Just a note that we from time to time will run automated scripts on the YAML, which will then remove this custom formatting.

Reply:
Yeah, I gave up adding double quotes everywhere after the previous change :)